Concrete_article_data.py

```diff
@@ -0,0 +1,54 @@
+# -*-encoding: utf-8 -
+
+
+import numpy as np
+import pandas as pd
+from sklearn import gaussian_process
+import matplotlib.pyplot as plt
+
+if __name__ == '__main__':
```

Comment:
One of the goal of this construct is to put everything in a funciton and avoid the potential issues with global code and especially global data... (the other goal is to make the file usable both as an imported module or an application). Thus the code should be something like:

```python
import sys
....

def main():
    Your current code

if __name__ == '__main__':
    sys.exit(main())
```